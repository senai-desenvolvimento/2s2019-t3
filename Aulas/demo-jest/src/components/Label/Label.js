import React, {Component} from 'react'

class Label extends Component{
    render() {
        return(
            <h1 className={this.props.className}>
              {this.props.texto}
            </h1>
        );
    }
}

export default Label;