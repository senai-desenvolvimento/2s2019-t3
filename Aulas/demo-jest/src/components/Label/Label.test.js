// Label.react.test.js
import React from 'react';
import Label from './Label';
import renderer from 'react-test-renderer';

test('Teste de titulo', () => {
  const titulos = [{ nome : "Fernando Henrique"}, {nome : "Gabriel Ramos"}];

  let titulo = titulos.map((titulo) => {
    <Label texto={titulo.nome}  className="teste"/>
  })

  const component = renderer.create(
    titulo
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

  console.log(tree);
});