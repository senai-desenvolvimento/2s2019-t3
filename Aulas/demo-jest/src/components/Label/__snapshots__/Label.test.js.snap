// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Teste de titulo 1`] = `null`;
